src/test/java/BaseTest.java

```diff
@@ -1,11 +1,72 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
 
+import java.time.Duration;
+
 public class BaseTest {
+    static WebDriver driver;
+    WebDriverWait wait;
+    public String url = "https://bbb.testpro.io/";
+
 
     @BeforeSuite
     static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
-}
\ No newline at end of file
+
+    @BeforeMethod
+    public void setUpBrowser() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--disable-notifications");
+        options.addArguments("--remote-allow-origins=*");
+        driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        wait = new WebDriverWait(driver, Duration.ofSeconds(4));
+        driver.manage().window().maximize();
+        driver.get(url);
+    }
+
+    @AfterMethod(alwaysRun = true)
+    static void tearDown() {
+        driver.quit();
+    }
+
+    public void login(String email, String password) {
+        enterEmail(email);
+        enterPassword(password);
+        clickLoginButton();
+    }
+
+    protected void enterPassword(String password) {
+        WebElement passwordInput = driver.findElement(By.cssSelector("[type='password']"));
+        passwordInput.click();
+        passwordInput.clear();
+        passwordInput.sendKeys(password);
+    }
+
+    protected void enterEmail(String email) {
+        WebElement emailInput = driver.findElement(By.xpath("//input[@type='email']"));
+        emailInput.click();
+        emailInput.clear();
+        emailInput.sendKeys(email);
+    }
+
+    protected void clickLoginButton() {
+        WebElement submitLoginButton = driver.findElement(By.cssSelector("button[type='submit']"));
+        submitLoginButton.click();
+    }
+
+    protected void deletePlaylistButton() {
+        WebElement deletePlaylistButton = driver.manage().timeots().ExplicitlyWait(millis:1000);
+
+    }
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/230178919-234cb644-8f7c-4b3b-a8ae-d831a42d149e.png)